docs/userguide/networking/dockernetworks.md

```diff
@@ -1,538 +0,0 @@
-<!--[metadata]>
```

Comment:
did you `git mv` this file, or "copy" and remove? currently it doesn't show that is was renamed from `dockernetworks.md` to `index.md`, and the git history is more difficult to find because of that